lib/super_good/solidus_taxjar.rb

```diff
@@ -36,6 +36,8 @@ def api
 
       def table_name_prefix
         "solidus_taxjar_"
+      def reporting
+        ::SuperGood::SolidusTaxjar::Reporting.new
```

Comment:
Looks like this might be the result of a bad merge but we're missing an `end` tag on the `table_name_prefix` method.